### PR TITLE
 Merge branch 'junmoy-update-tutorial' and resolve conflict

### PR DESCRIPTION
**Pull request by journey2w**
- Update TOC.md of Native in the PR.
- Change *-n.md to *.md , *-mn.md to *.-m.md, *-wn.md to *-w.md and *-w.md(for web app) to *.md in TOC.md.

### CODE REVIEW

File: docs/application/native/getting-started/mobile/first-app.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
 
 **Welcome to Tizen mobile native application development!**
 
 A mobile native application is created using the C language, and can be run on Tizen mobile devices. The application uses the native API, which provides various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
 
-Study the following instructions to help familiarize yourself with the Tizen [native application development
-process](../process/app-dev-process-n.md) as well as using the Tizen
-Studio and installing the created application on the emulator or target device. With the instructions, you can create and run a basic mobile native application, which displays some text on the screen with no user interaction:
+Study the following instructions to help familiarize yourself with the Tizen [native application development process](../../tutorials/process/app-dev-process-n.md) as well as using the Tizen Studio and installing the created application on the emulator or target device. With the instructions, you can create and run a basic mobile native application, which displays some text on the screen with no user interaction:
```
Comment (mijin85cho) [inline docs/application/native/getting-started/mobile/first-app.md]: Don't we need to change this `app-dev-process-n.md` into `app-dev-process.md`?

File: docs/application/native/getting-started/mobile/first-app.md [hunk 2]
```diff
@@ -28,119 +22,89 @@ Studio and installing the created application on the emulator or target device.
...
-    This step shows how you can make small alterations to the
-    application UI to improve the usability of your application.
+    This step shows how you can create the application UI and make small alterations to it to improve the usability of your application.
 
-When you are developing a more complex application, you can take
-advantage of the [native tools included in the Tizen
-Studio](../../../tizen-studio/native-tools/cover-native-n.md)
-to ease the tasks of creating functionality and designing the
-application UI.
+When you are developing a more complex application, you can take advantage of the [native tools included in the Tizen Studio](../../../tizen-studio/native-tools/cover-native-n.md) to ease the tasks of creating functionality and designing the application UI.
```
Comment (mijin85cho) [inline docs/application/native/getting-started/mobile/first-app.md]: Don't we need to change `cover-native-n.md` into `cover-native.md`?

File: docs/application/native/getting-started/mobile/first-app.md [hunk 3]
```diff
@@ -28,119 +22,89 @@ Studio and installing the created application on the emulator or target device.
...
 
 <a name="create"></a>
-## Creating a Project
-
+## Creating a Project {#create}
```
Comment (mijin85cho) [inline docs/application/native/getting-started/mobile/first-app.md]: `{#create}` is added.

File: docs/application/native/getting-started/mobile/first-app.md [hunk 4]
```diff
@@ -28,119 +22,89 @@ Studio and installing the created application on the emulator or target device.
...
 
-      You can fill the project name (3-50 characters) and the unique
-      package ID. You can also select the location and working sets by  clicking **More properties**.
+        You can enter the project name (3-50 characters) and the unique package ID. You can also select the location and working sets by clicking **More properties**.
 
-      ![Defining        properties](media/create_project_wizard_properties.png)
+        ![Defining properties](media/create_project_wizard_properties.png)
 
-      The Project Wizard sets up the project, creates the application        files using the default content from the template, and closes.        For more information on the Project Wizard and the available        templates, see [Creating Tizen Projects with Tizen Project        Wizard](../../../tizen-studio/native-tools/project-wizard-n.md).
+        The Project Wizard sets up the project, creates the application files using the default content from the template, and closes. For more information on the Project Wizard and the available templates, see [Creating Tizen Projects with Tizen Project Wizard](../../../tizen-studio/native-tools/project-wizard-n.md).
```
Comment (mijin85cho) [inline docs/application/native/getting-started/mobile/first-app.md]: Don't we need to change `project-wizard-n.md` into `project-wizard.md`?

File: docs/application/native/getting-started/mobile/first-app.md (hunk 3 above)
Comment (journey2w) [reply docs/application/native/getting-started/mobile/first-app.md]: Done

File: docs/application/native/getting-started/mobile/first-app.md (hunk 4 above)
Comment (journey2w) [reply docs/application/native/getting-started/mobile/first-app.md]: Done

File: docs/application/native/getting-started/mobile/first-app.md (hunk 1 above)
Comment (journey2w) [reply docs/application/native/getting-started/mobile/first-app.md]: Done.
However, I will update all files under getting-started directory next time. Please review this PR except the files under getting-started.

File: docs/application/native/getting-started/mobile/first-app.md (hunk 2 above)
Comment (journey2w) [reply docs/application/native/getting-started/mobile/first-app.md]: Done.
However, I will update all files under getting-started directory next time. Please review this PR except the files under getting-started.

**Pull request merged**